### DISCUSSION TITLE
If you open the project in another environment, dialogic will be initialized.

### DISCUSSION THREAD

**Discussion by PoliyasRoden** [dialogic-godot/dialogic#2120] [Q&A] [answered]
hello. Thank you for the continued development. Thanks to you, I am using it well.

If you open the project file on another computer, dialogic will be initialized. The interesting thing is that it runs properly. (The file is also maintained.)

It is impossible to check only inside godot. Is there a solution to this?

(It is Dialogic 2.X.)

Initialized appearance

![Initialized_appearance](https://github.com/dialogic-godot/dialogic/assets/122151351/e00d4254-ed3f-4e5c-99cd-31a94a3a41ae)

Looks like it runs well

![Looks_like_it_runs_well](https://github.com/dialogic-godot/dialogic/assets/122151351/d4da0525-6d77-4202-ac9d-cbc3bfa3b87d)

**Comment by CakeVR** [comment]
Hello, glad to hear you enjoy Dialogic! 

Is this correct: You want Dialogic inside the project but **not** initialise? 

In the **Project Settings**, you can uncheck (disable) Dialogic's `Enable`:
![image](https://github.com/dialogic-godot/dialogic/assets/65981767/33a74c4b-2e2f-465e-b4bf-96c6515132b6)

Please let me know, if that's the problem. I may have misunderstood you.

**Reply by Invertex** [reply]
I think what they mean is opening an already made project on another computer seems to revert Dialogic to its "fresh initialization" view.

Might be worth testing by passing a project around on the Discord. System language settings might come into play here too since I see the are writing Korean.

**Reply by PoliyasRoden** [reply]
Invertex is correct.

If you open an already created project on another computer, nothing will appear as if it has been initialized. (The timeline, characters, etc. I created are not visible)

However, when I ran the game, it appeared normally.

**Comment by Jowan-Spooner** [comment, marked as answer]
Late answer, but for anyone finding this later: This intended. 

The "opened resources" list is local to your computer, which is important for teams to be able to work together without constantly messing each others editor up. This means it's saved on your machine and not in the project. 

However no data is lost, you can always open character and timelines from the file system dock.
